### DISCUSSION TITLE
Is it required to have the hardware interface (Rotary/buttons/display) to try MiniDexed

### DISCUSSION THREAD

**Discussion by definable** [probonopd/MiniDexed#202] [Q&A]
First of all, great project and nice idea : -) 

At this moment I do not have time to construct some hardware, but I have an RPI CM4 which I can use. 

I was wondering if Minidexed can be used 'out of the box' as a USB MIDI device (e.g. with Ableton and a  MIDI keyboard). 
So without any additional  hardware (display/rotary) for setup/configuration.  I have no experience with synthesizers, so bare with me if this is a silly question. 

Maybe a related question, an RPI4 works, so the RPI CM4 works as well ?

**Comment by craigyjp** [comment]
You don't need the LCD or encoder to start off, but the usb ports are midi host, so you can connect a usb keyboard to the pi, but not directly from your daw, to do that you need a standard din midi or converter. You will hear the classic ROM 1A sounds but you will need to edit the ini config or several parts respond in Omni mode which is confusing at first.
